templates/html_report/_header.php

```diff
@@ -48,8 +48,21 @@
 
 
 <div class="page">
-    <div class="content">
+    <div class="content content-first">
         <div class="report-details">
             Created at <?php echo date('Y-m-d H:i:s'); ?>
         , with PHPMetrics <?php echo getVersion(); ?> (<a href="https://twitter.com/Halleck45">Jean-François Lépine</a>).
         </div>
+
+        <?php if([] !== $this->groups) {?>
+        <div class="content-full group-tabs">
+            <ul class="tabs">
+                <li class="<?php if(!$this->currentGroup) { echo 'active'; }?>"><a href="<?php echo $this->assetPath;?>index.html">All</a></li>
```

Comment:
```suggestion
                <li class="<?php if ('' === $this->currentGroup) { echo 'active'; }?>"><a href="<?php echo $this->assetPath;?>index.html">All</a></li>
```

Reply:
I'm generally in favor if stric equality, but a simple `!` often avoid bugs (here currentGroup is nullable)